
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="src/favicon.ico">

    <title>Gregory Ditzler (The University of Arizona) - Research</title>

    <!-- Bootstrap core CSS -->
    <link href="dist/css/ua-bootstrap.css" rel="stylesheet">
    <!-- Bootstrap theme -->
    <link href="dist/css/ua-bootstrap-theme.min.css" rel="stylesheet">

    <!-- Custom styles for this template -->
    <link href="theme.css" rel="stylesheet">

    <!-- Just for debugging purposes. Don't actually copy these 2 lines! -->
    <!--[if lt IE 9]><script src="../../assets/js/ie8-responsive-file-warning.js"></script><![endif]-->
    <script src="../../assets/js/ie-emulation-modes-warning.js"></script>

    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
  </head>

  <body role="document">
    <!--   ++++++++++++++ Place inside body tag ++++++++++++++   -->
  <header id="region_header_ua" class="l-arizona-header bg-cochineal-red">
    <section class="container l-container">
      <div class="row">
        <a href="http://www.arizona.edu" title="The University of Arizona homepage" class="arizona-logo">
           <img alt="The University of Arizona Wordmark Line Logo White" src="https://brand.arizona.edu/sites/default/files/digital-resources/ua_wordmark_line_logo_white_rgb.svg" class="arizona-line-logo">
        </a>
       </div>
     </section>
  </header>




    <!-- Fixed navbar -->
    <nav class="navbar navbar-inverse navbar-fixed-top">
      <div class="container">
        <div class="navbar-header">
          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
          <!--<a class="navbar-brand" href="#">Gregory Ditzler</a>-->
        </div>
        <div id="navbar" class="navbar-collapse collapse">
          <ul class="nav navbar-nav">
            <li><a href="index.html">Home</a></li>
            <li><a href="about.html">About</a></li>
            <li><a href="news.html">News</a></li>
            <li><a href="misc.html">Miscellaneous</a></li>
            <li><a href="publications.html">Publications</a></li>
            <li class="active"><a href="research.html">Research</a></li>
            <li><a href="teaching.html">Teaching</a></li>
            <li><a href="people.html">People</a></li>
          </ul>

        </div>
      </div>
    </nav>


    <div class="container theme-showcase" role="main">

      <!-- Main jumbotron for a primary marketing message or call to action -->
      <div>
      <a href="http://www.ece.arizona.edu"><img src="http://www.ece.arizona.edu/sites/all/themes/danland/images/slideshows/engr10.jpg" align="middle" alt="University of Arizona" width="1130px"></a>
      </div>

      <div class="page-header">
        <h1>Research Overview</h1>
      </div>
      My research interests can best be described as applied machine learning
      and data mining. Refer to my list of publications for more details about
      the research works that are briefly highlighted on this webpage. My areas
      of research include:
      <ul>
        <li>Large scale feature subset selection</li>
        <li>Incremental and online learning in nonstationary environments</li>
        <li>Applied machine learning to comparative metagenomics</li>
      </ul>

      <!-- ***************************************************************** -->
      <div class="page-header">
        <h1>Large Scale Feature Subset Selection</h1>
      </div>
      <p>
        There are an ever increasing number of applications that are generating massive amounts of data, which are of high dimensionality. Not only is the cardinality of the data rapidly increasing, but also the dimensionality. Such applications include analysis of a vast amount of data generated by social networks, media networks, blogs, healthcare informatics and genomics to name a few. Unfortunately, not all of the features in the data informative or meaningful and we are completely unaware of which features are meaningful. Therefore, we need algorithms to extract the meaningful and informative features, while remaining scalable to a massive population of data.
      </p>
      <div class="panel panel-info">
        <div class="panel-heading">
          <h3 class="panel-title">Highlighted Publications</h3>
        </div>
        <div class="panel-body">
          <ol>
            <li>G. Ditzler, R. Polikar, and G. Rosen, “<a href="../publications/tnnls2015.pdf">A bootstrap based Neyman-Pearson test for identifying variable importance</a>,” IEEE Transactions on Neural Networks and Learning Systems, 2015, vol. 26, no. 4, pp. 880-886.</li>
            <li>G. Ditzler, M. Austen, R, Polikar, and G. Rosen, “<a href="../publications/cidm2014.pdf">Scalable Subset Selection and Variable Importance</a>,” IEEE Symposium on Computational Intelligence in Data Mining, 2014. (travel award)</li>
          </ol>
        </div>
      </div>




      <!-- ***************************************************************** -->
      <div class="page-header">
        <h1>Incremental &amp; Life Long Learning with Ensembles</h1>
      </div>
      <p>
        Two of the more common assumptions that applied machine learning researchers make when using an algorithm is that: (1) the training &amp; testing data are sampled from a fixed - albeit unknown - probability distribution, and (2) there are an equal number of samples from all classes. The former is referred to as concept drift (a.k.a., learning in non-stationary environments) when new data are presented over times, and the latter is known as class imbalance. I developed two incremental multiple classifier system solutions, namely Learn$++.NIE and Learn++.CDS, for explicitly addressing the  joint problem of learning from data streams with concept drift and class imbalance, which has been largely understudied in the literature. I have demonstrated that these approaches are quite useful in practice and they beat the state-of-the-art algorithms in terms of skew insensitive statistics, which are not a simple error calculation. I have also developed algorithms for learning new - potentially imbalanced - classes with an ensemble of classifiers.
      </p>
      <div class="panel panel-success">
        <div class="panel-heading">
          <h3 class="panel-title">Highlighted Publications</h3>
        </div>
        <div class="panel-body">
          <ol>
            <li>G. Ditzler, M. Roveri, C. Alippi, and R. Polikar, “<a href="publications/ieee-cim2015.pdf">Adaptive strategies for learning in nonstationary environments: a survey</a>,” IEEE Computational Intelligence Magazine, 2015, Accepted.</li>
            <li>G. Ditzler and R. Polikar, “<a href="publications/tkde2013.pdf">Incremental learning of concept drift from streaming imbalanced data</a>,” in IEEE Transactions on Knowledge &amp; Data Engineering, 2013, vol. 25, no. 10, pp. 2283–2301.  </li>
            <li>G. Ditzler, G. Rosen, and R. Polikar, “<a href="publications/ijcnn2014.pdf">Domain Adaptation Bounds for Multiple Expert Systems Under Concept Drift</a>,” International Joint Conference on Neural Networks, 2014. (travel award &amp; best student paper)</li>
          </ol>
        </div>
      </div>


      <!-- ***************************************************************** -->
      <div class="page-header">
        <h1>Applied Machine Learning in Life Sciences</h1>
      </div>
      <p>
        Metagenomics is the study of genetic material obtained directly from an environmental sample, which means that everything is sequenced from a sample (i.e., all of the organisms). Note that this differs from traditional genomics in that a single genome is generally sequenced. One of the important aspects of metagenomic sequencing is that it allows researchers to obtain an understanding of the bacterial and functional profiles of an environment from where the sample was collected. These profiles are important because the bacteria have been hypothesized to play a crucial role in not only human health, but also our surrounding environments.

        I have applied my expertise on feature subset selection to 16S and metagenomic data to help microbial ecologists determine the protein families and microorganisms that best differentiate between multiple phenotypes within an environmental study. I worked closely with a colleague to implement information-theoretic feature subset selection algorithms in QIIME, which is a heavily used analysis tool in microbial ecology, and standalone tool that are freely available without restriction to researchers interested in using our tools (see <a href="http://goo.gl/YAnxRD">http://goo.gl/YAnxRD</a>).
      </p>
      <div class="panel panel-warning">
        <div class="panel-heading">
          <h3 class="panel-title">Highlighted Publications</h3>
        </div>
        <div class="panel-body">
          <ol>
            <li>G. Ditzler, R. Polikar, and G. Rosen, “<a href="publications/tnbs2015.pdf">Multi-Layer and Recursive Neural Networks for Metagenomic Classification</a>,” IEEE Transactions on Nanobioscience, 2015, Accepted.</li>
            <li>G. Ditzler, Y. Lan, J.-L. Bouchot, and G. Rosen, “<a href="http://link.springer.com/referencework/10.1007%2F978-1-4614-6418-1">Feature selection for metagenomic data analysis</a>,” Encyclopedia of Metagenomics, 2014. </li>
            <li>J.-L. Bouchot, W. Trimble, G. Ditzler, Y. Lan, S. Essinger, and G. Rosen, “<a href="http://store.elsevier.com/Computational-Systems-Biology/isbn-9780124059269/">Advances in machine learning for processing and comparison of metagenomic data</a>,” Computational Systems Biology, Springer, 2014.</li>
            <li>G. Ditzler and G. Rosen, “<a href="publications/bigls2014.pdf">Feature Subset Selection for Inferring Relative Importance of Taxonomy</a>,” ACM International Workshop on Big Data in Life Sciences, 2014. (invited and travel award)</li>
          </ol>
        </div>
      </div>



      <br>
      <br>
      <br>

    </div> <!-- /container -->

    <footer>
      <center>
        <small> Copyright 2015 © Arizona Board of Regents.</small>
      </center>
    </footer>


    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
    <script src="dist/js/bootstrap.min.js"></script>
    <script src="assets/js/docs.min.js"></script>
    <!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
    <script src="../../assets/js/ie10-viewport-bug-workaround.js"></script>
  </body>
</html>
